core/src/com/unciv/ui/overviewscreen/GlobalPoliticsOverviewTable.kt

```diff
@@ -170,7 +170,7 @@ class GlobalPoliticsOverviewTable (
     }
 
     private fun getCivName(otherciv: CivilizationInfo): String {
-        if (viewingPlayer.knows(otherciv)) {
+        if (viewingPlayer.knows(otherciv) && otherciv.civName != viewingPlayer.civName) {
```

Comment:
Surely 'or'?

Reply:
sorry, I'm not quite sure I understand. You're saying this should be an OR instead of AND?

Reply:
Yes, because currently, it will show 'unknown' it the city state's ally is the viewing player

Reply:
ohhh yes I wasn't sure why it would be fine for the rest but I forgot the other lines have the .knows inverted :))